datavault-broker/src/main/resources/import.sql

```diff
@@ -133,3 +133,19 @@ insert into GroupOwners (group_id, user_id) values ('MHSE', 'user3');
 insert into GroupOwners (group_id, user_id) values ('SHSS', 'user4');
 
 insert into Clients (id, name, apiKey, ipAddress) values ('datavault-webapp', 'Datavault Webapp', 'datavault-webapp', '127.0.0.1');
+
+insert into Permissions (id, permission, label, type) values ('VAULT_PERMISSION_1', 'VAULT_PERMISSION_1', 'Vault Permission 1', 'VAULT');
```

Comment:
This is fine for the sake of testing but I'm not fully happy with it. We need one entry per permission so with this approach of manually running an import script whenever a new environment comes up, we'd also need to manually run an update to add an entry to this table whenever a new permission was created. I'm going to change this such that the `Permission` enum has all the default values for these fields, and then when the application comes up, we add any new permissions and remove any that aren't present in the enum